public-ui/src/app/views/survey-view/survey-view.component.ts

```diff
@@ -200,10 +203,33 @@ export class SurveyViewComponent implements OnInit, OnDestroy {
     }
   }
 
-  public showAnswerGraphs(q, a: AchillesResult) {
-    q.selectedAnswer = a;
+  public showAnswerGraphs(a: any) {
+    if (a.expanded) {
+      a.expanded = false;
+      return;
+    }
+    a.expanded = true;
+  }
```

Comment:
nit: I would put a newline between methods

Reply:
strange but i cannot add space between the methods. Not sure if i can change the lint rule somewhere.